README.md

```diff
@@ -8,6 +8,60 @@ Some of these automations are provided as [Reusable workflows](https://docs.gith
 For these workflows, you can embed them in a workflow you run at the `job` level.
 Examples are provided below.
 
+## Lint
+
+The lint workflow installs and runs the relevant linters for the repository. It expects the following
+`make` targets:
+
+- `setup-lint`: Installs relevant linters (only needs to work on Ubuntu)
+- `lint`: Runs relevant linters
+
+### Usage
+
+An example workflow:
+
+```yaml
+name: QA
+on:
+  push:
+    branches:
+      - "main"
+      - "feature/*"
+      - "hotfix/*"
+      - "release/*"
+      - "renovate/*"
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: lengau/starflow/lint@work/CRAFT-3602/test-workflows
```

Comment:
```suggestion
      - uses: canonical/starflow/lint@work/CRAFT-3602/test-workflows
```

Reply:
This needs to become just:

```suggestion
      - uses: canonical/starflow/lint@main
```

before merging. But it'll break when I make that change until the merge occurs